13-loops/exercises/02-sum-the-numbers-verbose/main.go

```diff
@@ -26,4 +20,15 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	var sum int
+
+	for i := 1; i <= 10; i++ {
```

Comment:
when you are using a value two or more times like 10 here, once in loop and then in the if statement later, then prefer to define it as const/var and then use that constant. This way later if you want to modify the max value then you need to modify only once and also readability is better this way.

Reply:
Fixed